password_cracker.md

```diff
@@ -15,6 +15,14 @@ wikibook:
   - "Synchronization, Part 5: Condition Variables"
   - "Synchronization, Part 6: Implementing a barrier"
 ---
+## Backstory
+After finally being taken off of PIP due to your outstanding malloc performance you can finally relax a little. So take a Pomelo ElCroix out of the minifridge and relax!
+
+Don't lower your standing desk just yet though because as it turns out, while you were competing an intern deleted the plaintext file containing everyone's passwords and now

```

Comment:
```suggestion
Don't lower your standing desk just yet, because as it turns out, while you were competing, an intern deleted the plaintext file containing everyone's passwords.
```